selftests/unit/test_loader.py

```diff
@@ -50,6 +50,45 @@ def test(self):
     main()
 """
 
+AVOCADO_TEST_TAGS = """#!/usr/bin/env python
+from avocado import Test
+from avocado import main
+
+import time
+
+class DisabledTest(Test):
```

Comment:
Mainly the snippets were recommended for ugly code to avoid people copying it...